bigquery/src/main/java/com/google/cloud/bigquery/samples/BigqueryServiceFactory.java

```diff
@@ -30,16 +30,14 @@
 import java.util.Collection;
 
 /**
- * This class creates our Service to connect to Bigquery including auth.
+ * This class creates our Service to connect to BigQuery including auth.
  */
-public final class BigqueryServiceFactory {
+public final class BigQueryServiceFactory {
```

Comment:
Sigh. Yes. I'm having trouble getting git to accept that the filename is changed (I think because I'm on my Mac)